tcms/testruns/templates/testruns/get.html

```diff
@@ -327,8 +328,40 @@ <h4 class="modal-title" id="add-hyperlink-modal-title">{% trans "Add hyperlink"
   </div>
 </div>
 
+<div class="modal fade" id="one-click-bug-report-modal" tabindex="-1" role="dialog" aria-labelledby="one-click-bug-report-title" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true" aria-label="Close">
+          <span class="pficon pficon-close"></span>
+        </button>
+        <h4 class="modal-title" id="one-click-bug-report-title">{% trans "Report bug" %}</h4>
+      </div>
+      <form class="form-horizontal one-click-bug-report-form">
+        <div class="modal-body">
+          <div class="form-group">
+            <label class="col-sm-3 control-label" for="issue-tracker-select">{% trans "Issue Tracker" %}</label>
+            <div class="col-sm-9">
+                <select name="issue-tracker" id="id-issue-tracker" class="form-control selectpicker">
+                    {% for btr in bug_trackers %}
+                        <option value="{{ btr.pk }}">{{ btr.name }}</option>
+                    {% endfor %}
+                </select>
+            </div>
+          </div>
+        </div>
+        <div class="modal-footer">
+            <button type="button" class="btn btn-danger" data-dismiss="modal">{% trans "Cancel" %}</button>
+            <button type="submit" class="btn btn-primary one-click-bug-report-button">{% trans "Report" %}</button>
```

Comment:
Use "Report bug" here as well b/c "Report" is used with other meaning and that makes a mess of translations.